databases/week1/DATABASES_WEEK1_HOMEWORk/index.sql

```diff
@@ -0,0 +1,69 @@
+
+
+-- Find out how many tasks are in the task table
+SELECT COUNT(*) FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) FROM task WHERE task.due_date is null;
+
+-- Find all the tasks that are marked as done
+SELECT 
+	task.*, 
+    status.name 
+FROM task 
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name = 'done';
+
+-- Find all the tasks that are not marked as done
+SELECT 
+	task.*, 
+    status.name 
+FROM task 
+INNER JOIN status on task.status_id = status.id 
+WHERE status.name != 'done';
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT * FROM task ORDER BY created DESC;
+
+-- Get the single most recently created task
+SELECT * FROM task ORDER BY created DESC LIMIT 1;
+
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT 
+    id,
+    title, 
+    description,
+    due_date 
+FROM task 
+WHERE title REGEXP 'database' OR description REGEXP 'database';
```

Comment:
While you can use regular expressions in SQL you should always prefer the LIKE operator whenever possible, since it will almost always be faster.